styles.css

```diff
@@ -1,102 +1,198 @@
-    padding: 8px 12px;
-    border-radius: 4px;
-    color: rgb(201, 170, 226);
-    box-shadow: 5px 4px 6px rgba(29, 3, 53, 0.8); 
-  }
-
-  .coverimg {
-    border-radius: 30px 190px 80px 190px;
-    box-shadow: 0px 0px 30px 1px;
-    display: flex;
-    margin-left: auto;
-    margin-right: auto;
-    width: auto;
-  }
-
-  .centre {
-    text-align: center;
-    margin-left: auto;
-    margin-right: auto;
-    width: 100%;
-  }
-
-  .buttontext {
-    background-color: rgba(159, 121, 190, 0.7);
-    padding: 1px;
-    border-radius: 4px;
+  box-sizing: border-box;
+}
+
+html {
+  background-image: linear-gradient(to bottom, rgba(48, 19, 117, 0.651), rgba(202, 180, 198, 0.651));
+}
+
+body {
+  font-family: Helvetica, Arial, sans-serif;
+  font-size: 110%;
+  line-height: 1.5;
+  margin: 1.5em;
+}
+
+h1 {
+  font-family: "Bradley Hand", Helvetica, Arial, sans-serif;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  line-height: 1.2;
+  text-decoration: underline;
+}
+
+hr{
+  border: 0;
+  width: auto;
+  height: 1px;
+  background-color: rgba(72, 66, 83, 0.459);
+}
+
+.container-head {
+border: 1px solid none;
+display: flex;
+flex-flow: row wrap;
+justify-content: flex-start;
+align-items: center;
+gap: 50px;
+}
+
+.child-head-name {
+border: 1px solid none;
+flex: 1;
+}
+
+.child-head-nav {
+border: 1px solid none;
+}
+
+.main-nav {
     border: 1px solid none;
-  }
-
-  a[href^="mailto"]::before {
-    content: "📧 ";
-  }
-
-  a[href^="tel"]::before {
-    content: "📞";
-  }
-  
-  .contact {
-    padding: 2px;
-    margin: 10px;
-  }
\ No newline at end of file
+    height: 50px;
+    padding: 4px;
+    display: flex;
+    gap: 50px;
+    align-items: center;
+    flex-flow: row wrap;
+}
+
+.main-nav > * {
+  flex-grow: 1;
+  flex-basis: 0%;
+}
+
+.button {
+  background-color: rgba(96, 62, 136, 0.822);
+  padding: 8px 12px;
+  border-radius: 4px;
+  color: rgb(201, 170, 226);
+  box-shadow: 5px 4px 6px rgba(29, 3, 53, 0.8); 
+}
+
+#coverimg {
+  border-radius: 30px 190px 80px 190px;
+  box-shadow: 0px 0px 30px 1px;
+  display: flex;
+  margin-left: auto;
+  margin-right: auto;
+  width: auto;
+}
+
+figcaption {
+  text-align: right;
+}
+
+.container-sum {
+  border: 1px solid none;
+  display: flex;
+  gap: 10px;
+  flex-flow: row wrap;
+  justify-content: center;
+  align-items: center;
+  align-content: stretch;
+}
+
+.child-sumpic {
+border: 1px solid none;
+}
+
+.child-sumtxt {
+border: 1px solid none;
+padding: 8px 12px;
+flex: 1;
+}
+
+img {
+  max-height: 100%;
+  min-width: 100%;
+}
+
+.no-bullets {
+list-style-type: none; 
+}
+
+.buttontext {
+  background-color: rgba(176, 146, 201, 0.7);
+  padding: 1px 10px;
+  border-radius: 5px;
+  color: rgb(98, 58, 131);
+  border: 1px solid none;
+}
+
+.container-contact {
+  border: 1px solid none;
+  gap: 5px;
+  display: flex;
+  flex-flow: column wrap;
+  justify-content: space-evenly;
+  align-items: center;
+  align-content: center;
+}
+
+.child-contactadr{
+  border: 1px solid none;
+  gap: ;
```

Comment:
just a reminder to the missing value, maybe in the layout gap won' t be necessary 😃

Reply:
Thank you! :)